extensions/ql-vscode/src/databases/local-databases-ui.ts

```diff
@@ -684,7 +685,7 @@ export class DatabaseUI extends DisposableObject {
     );
   }
 
-  private async handleClearCache(): Promise<void> {
+  private async handleClearCache(mode: "trim" | "clear"): Promise<void> {
```

Comment:
It would be nice to extract `"trim" | "clear"` into a `type` so it can be reused below.